### PR TITLE
feature: add node package managers poc

### PR DESCRIPTION
**Pull request by mdominguez56**
# What's this PR do?

In this PR the goal is to show information about different alternatives for package management in Node. And with that information you can make a decision about which one is more convenient to use.

I updated the readme inside the `/examples` folder, so that the new information added there is also reflected. I don't know if that is correct or not for me to do it.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -108,6 +109,11 @@ normal React hooks to State management libraries. Check the full list with more
 - [MobX State Tree](./state-management/examples/bidirectional-mobx-state-tree/)
 - [React Easy State](./state-management/examples/bidirectional-easy-state/)
 
+## Node Package Managers
```
Comment (ulises-jeremias) [inline examples/README.md:112]: Can we move this at the very beginning of the examples list? this analysis is more important than the rest 🥳

File: examples/node-package-managers/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+![Logo](https://pnpm.io/fr/img/pnpm-no-name-with-frame.svg)
+
+# PNPM POC
```
Comment (ulises-jeremias) [inline examples/node-package-managers/README.md:3]:
```suggestion
# Node Package Managers Comparison
```

File: examples/node-package-managers/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+When installing dependencies with npm or Yarn Classic, all packages are hoisted to the root of the modules directory. As a result, source code has access to dependencies that are not added as dependencies to the project.
+
+By default, pnpm uses symlinks to add only the direct dependencies of the project into the root of the modules directory.
+
+## Installation
+
+Install pnmpm with npm
+
+```bash
+  npm install -g pnpm
```
Comment (ulises-jeremias) [inline examples/node-package-managers/README.md:36]:
```suggestion
npm install -g pnpm
```

File: examples/node-package-managers/README.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+## Filtering
+
+Filtering allows you to restrict commands to specific subsets of packages.
+
+pnpm supports a rich selector syntax for picking packages by name or by relation.
+
+Selectors may be specified via the --filter (or -F) flag:
+
+```bash
+  pnpm --filter <package_selector> <command>
```
Comment (ulises-jeremias) [inline examples/node-package-managers/README.md:86]:
```suggestion
pnpm --filter <package_selector> <command>
```

File: examples/node-package-managers/README.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+Here are a few of the most popular open source projects that use the workspace feature of pnpm:
+
+![workspaces](./images/pnpm-workspaces-examples.png)
+
+To learn more about the implementation of workspaces, you can see the following link: _https://pnpm.io/workspaces_
+
+## License
+
+[MIT](https://choosealicense.com/licenses/mit/)
```
Comment (ulises-jeremias) [inline examples/node-package-managers/README.md:105]: we can remove the license from here since we already have it at the root

**Review by ulises-jeremias — COMMENTED** [review]
just some changes, the rest looks great!

File: examples/README.md (hunk 1 above)
Comment (mdominguez56) [reply examples/README.md]: Yes for sure! I already moved it.

File: examples/node-package-managers/README.md (hunk 2 above)
Comment (mdominguez56) [reply examples/node-package-managers/README.md]: Done

File: examples/node-package-managers/README.md (hunk 3 above)
Comment (mdominguez56) [reply examples/node-package-managers/README.md]: Done

File: examples/node-package-managers/README.md (hunk 4 above)
Comment (mdominguez56) [reply examples/node-package-managers/README.md]: Done

File: examples/node-package-managers/README.md (hunk 5 above)
Comment (mdominguez56) [reply examples/node-package-managers/README.md]: Done!

**Review by ulises-jeremias — APPROVED** [review]

**Pull request merged**